app/models/server_puppet_data.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class ServerPuppetData
+  attr_reader :client
+
+  def initialize
+    @client = PuppetdbClient.new(Settings.PUPPETDB_ENDPOINT)
```

Comment:
We currently configure the GitHub client directly in the github_client class. For sake of consistency, can we move this call to the config gem to the client class?